### PR TITLE
[REEF-1150] Use `static` members properly by removing access by `instance` reference

### PR DESCRIPTION
**Pull request by dongjoon-hyun**
JIRA:
  [REEF-1150](https://issues.apache.org/jira/browse/REEF-1150)

Pull Request:
  This closes #

### CODE REVIEW

File: lang/java/reef-common/src/main/java/org/apache/reef/util/logging/LoggingScopeFactory.java [hunk 1]
```diff
@@ -119,7 +119,7 @@ public LoggingScope driverStart(final StartTime startTime) {
    * @return
    */
   public LoggingScope driverStop(final long timeStamp) {
-    return new LoggingScopeImpl(LOG, logLevel, this.DRIVER_STOP + " :" + timeStamp);
+    return new LoggingScopeImpl(LOG, logLevel, DRIVER_STOP + " :" + timeStamp);
```
Comment (afchung) [inline lang/java/reef-common/src/main/java/org/apache/reef/util/logging/LoggingScopeFactory.java]: Could you also fix the `" :"` to be `": "`?

**Comment by afchung** [comment]
Looks good to me.

File: lang/java/reef-common/src/main/java/org/apache/reef/util/logging/LoggingScopeFactory.java (hunk 1 above)
Comment (dongjoon-hyun) [inline lang/java/reef-common/src/main/java/org/apache/reef/util/logging/LoggingScopeFactory.java]: Oh, sure!

**Comment by dongjoon-hyun** [comment]
Thank you, @afchung .  I updated the spacing, too.

**Comment by tcNickolas** [comment]
I'll ask the same question as I asked in JIRA here: is there a way to detect this kind of code automatically? It's a good change, but it's better if it's automated.

**Comment by dongjoon-hyun** [comment]
Oh, I will answer in JIRA. I missed your comment.

**Comment by dongjoon-hyun** [comment]
Thank you, @afchung !

**Pull request closed without merging**